src/QMCWaveFunctions/lcao/AOBasisBuilder.h

```diff
@@ -142,7 +142,8 @@ bool AOBasisBuilder<COT>::put(xmlNodePtr cur)
 
   // Numerical basis is a special case
   if (basisType == "Numerical")
-    radFuncBuilder.openNumericalBasisH5(cur);
+    APP_ABORT("Purely numerical atomic orbitals are not supported any longer.");
```

Comment:
Here the build is derived from MPIObjectBase so myComm is available.
Because all the ranks hits here, let us do `myComm->barrier_and_abort()`

Reply:
Done